app/controllers/foods_controller.rb

```diff
@@ -18,6 +18,14 @@ def create
     end
   end
 
+  def destroy
+    @food = Food.find(params[:id])
+    authorize! :destroy, @food
+    @food.destroy
+    flash[:success] = 'Food is removed successfully'
```

Comment:
Hey @ridaarif98,

I think here you can add a case when the user is not authorized or for some other reason, they are unable to delete. This way the app wont break.

Reply:
@faizi2500 thank you for the review 😄 . I will try to implement the changes